osf/external/spam/tasks.py

```diff
@@ -10,14 +10,14 @@
 DOMAIN_REGEX = re.compile(r'(?P<protocol>\w+://)?(?P<www>www\.)?(?P<domain>[\w-]+\.\w+)(?P<path>/\w*)?')
 
 @celery_app.task()
-def reclassify_domain_references(notable_domain_id, previous_note):
+def reclassify_domain_references(notable_domain_id, current_note, previous_note):
     from osf.models.notable_domain import DomainReference, NotableDomain
     domain = NotableDomain.load(notable_domain_id)
     references = DomainReference.objects.filter(domain=domain)
     with transaction.atomic():
         for item in references:
-            item.is_triaged = domain.note != NotableDomain.Note.UNKNOWN
```

Comment:
For some reason, when I put a debugger inside the celery task, `domain.note` is still the value before the save. This fix uses value passed to the task instead of relying on whatever the value is fetched from db.